opennlp-tools/src/test/java/opennlp/tools/tokenize/TokenizerFactoryTest.java

```diff
@@ -194,7 +199,10 @@ void checkCustomPatternForTokenizerME(String lang, String pattern, String senten
     String[] tokens = tokenizer.tokenize(sentence);
 
     Assertions.assertEquals(expectedNumTokens, tokens.length);
-    String[] sentSplit = sentence.replaceAll("'", " '").split(" ");
+    String[] sentSplit = sentence
+            .replaceAll("'", " '")
+            .replaceAll(",", " ,")
```

Comment:
One of my examples failed returning a token as `word,`, so I added this extra replace in this test :+1: